include/morphio/soma.h

```diff
@@ -65,15 +65,10 @@ class Soma
      */
     floatType maxDistance() const;
 
-  private:
     explicit Soma(const std::shared_ptr<Property::Properties>&);
```

Comment:
I believe that it was done this way b/c the public interface is smaller; which means less potentially breaking changes and ppl depending on certain implementation details. (ie: Hyrum's Law); that's why I'd lean to not make it public.

Reply:
However, resorting to using friends to bypass the private constructor seems like an unnecessary complexity to me, isn't it?